docs/README.raspberrypi

```diff
@@ -38,7 +38,7 @@ build for the first Raspberry Pi, the commands have to be adapted to use
        --with-toolchain=$RPI_DEV/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf \
        --with-firmware=$RPI_DEV/firmware \
        --with-platform=raspberry-pi2 \
-       --build=i686-linux \
+       --build=x86_64-linux-gnu \
```

Comment:
This argument isn't needed anymore.
It was necessary as the rbpi toolchain had i686 only executables at the beginning.

Reply:
Let me check if it builds without it.
If memory serves some packages were picky about not having both --host and --build set.